src/include/ipc/dai-amd.h

```diff
@@ -23,5 +23,6 @@ struct sof_ipc_dai_acp_params {
 	uint32_t reserved0;
 	uint32_t fsync_rate;
 	uint32_t tdm_slots;
+	uint32_t tdm_mode;
```

Comment:
is this needed? You already have a format at a higher level that you can use

````
#define SOF_DAI_FMT_DSP_A	4 /**< L data MSB after FRM LRC */
#define SOF_DAI_FMT_DSP_B	5 /**< L data MSB during FRM LRC */
````

that would avoid an ABI change

Reply:
Thanks for the suggestion, we will look into it.

Reply:
@maruthi-machani any update here ? Still needed or will existing ABI work ?

Reply:
updated topology to get format. removed abi changes.